testsuite/features/secondary/trad_deleted_migrate_to_minion.feature

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# This test case does not test a valid scenario. In SUMA you normally do not
+# delete a traditional client and then register it again as Salt minion. You
+# would always just bootstrap the traditional client as a Salt minion again
+# and Salt would recognize that it is a traditional client and will take the
+# the necessary steps to migrate it.
+
+@sle_client
+@scope_traditional_client
+Feature: Migrate a unregistered traditional client into a Salt minion
+  As an authorized user
+  I want to bootstrap a Salt minion after unregistering a traditional client
+
+  Scenario: Migrate a SLES client into a Salt minion
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Systems > Bootstrapping"
+    And I enter the hostname of "sle_client" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select "1-SUSE-KEY-x86_64" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+
+  Scenario: Wait until the Salt minion appears
+    When I wait until onboarding is completed for "sle_client" salt minion
+
+  Scenario: Check that the migrated system is now a minion
```

Comment:
This scenario name is already used in `trad_migrate_to_minion.feature`

Ideally, it would be better to use globally unique scenario names (that do not exist yet)

Reply:
I appended the context at the end of each scenario.